apps/test-suite/tests/Image.js

```diff
@@ -8,13 +8,27 @@ import { mountAndWaitFor, mountAndWaitForWithTimeout, TimeoutError } from './hel
 
 export const name = 'Image';
 
-const REMOTE_SOURCE = { uri: 'http://source.unsplash.com/random' };
+const REMOTE_SOURCE = { uri: 'https://source.unsplash.com/random' };
+const REMOTE_KNOWN_SOURCE = {
+  uri: 'https://images.unsplash.com/photo-1701743805362-86796f50a0c2?w=1080',
+  blurhash: 'LPCGGRxa9GWB01WBs:R*_2ayV@WB',
+};
 const NON_EXISTENT_SOURCE = { uri: 'file://non_existent_path.jpg' };
 const ANIMATED_IMAGE_SOURCE = {
   uri: 'https://media1.giphy.com/media/gZEBpuOkPuydi/giphy.gif?cid=ecf05e47fc23hje74g3ryyry6xnui81pej12o4eojtd9ruax&ep=v1_gifs_search&rid=giphy.gif&ct=g',
 };
 
 export async function test(t, { setPortalChild, cleanupPortal }) {
+  const throws = async (run) => {
+    let error = null;
+    try {
+      await run();
+    } catch (e) {
+      error = e;
+    }
+    t.expect(error).toBeTruthy();
+  };
+
```

Comment:
It's not needed if you apply previous suggestion
```suggestion
```

Reply:
see comment https://github.com/expo/expo/pull/26430#discussion_r1453267874